modules/user/user.resolver.js

```diff
@@ -7,14 +7,15 @@ const userQuery = {
 };
 const userMutation = {
   registerUser: (parent, args) => userService.registerUser(args.user),
-  loginUser: (parent, args) => userService.loginUser(args.user),
+  loginUser: (parent, args) => userService.loginUser(args.user, args.language),
   deleteUser: (parent, args) => userService.deleteUser(args.id),
   updateUserById: (parent, args, context) => (context.user
     ? userService.updateUserById(args.user, args.id)
     : new Error('Unauthorized')),
   updateUserByToken: (parent, args, context) => (context.user
-    ?  userService.updateUserByToken(args.user, context.user)
+    ? userService.updateUserByToken(args.user, context.user)
     : new Error('Unauthorized')),
+  confirmUser: (parent, args) => userService.confirmUser(args.token, args.language),
```

Comment:
Revert this change, please, we are going to send back unlocalized error messages and localize them only on backend

Reply:
removed